docs/content/en/docs/concepts/eksapackages.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "EKS Anywhere curated packages"
+linkTitle: "EKS Anywhere curated packages"
+weight: 4
+date: 2022-04-12
+description: >
+  Explanation of the process of managing curated packages in an EKS Anywhere cluster
+---
+
+EKS Anywhere curated packages is a framework to manage installation, configuration and maintenance of components that provide general operational capabilities for Kubernetes applications. Examples of these components include CSI, Ingress, LoadBalancer, Container Registry, Service Mesh, Secret Management, etc.
```

Comment:
I don't think `framework` is the correct word here. More like a tool, or even a controller if we want to be very specific.

Reply:
If we don't mind repeating `packages`, we'd say it's a `package manager`. Alternatively: `EKS Anywhere [curated] packages is a management system for installation, configuration and maintenance of additional components for your Kubernetes cluster.`